src/applications/yellow-ribbon/00-required.e2e.spec.js

```diff
@@ -0,0 +1,122 @@
+// Node modules.
+import toLower from 'lodash/toLower';
+// Relative imports.
+import E2eHelpers from 'platform/testing/e2e/helpers';
+import Timeouts from 'platform/testing/e2e/timeouts.js';
+import createMockEndpoint from 'platform/testing/e2e/mock-helpers';
+import manifest from './manifest.json';
+import stub from './constants/stub.json';
+import {
+  deriveDegreeLevel,
+  deriveDivisionProfessionalSchool,
+  deriveEligibleStudentsLabel,
+  deriveLocationLabel,
+  deriveMaxAmountLabel,
+  deriveNameLabel,
+} from './components/SearchResult';
```

Comment:
Some engineers prefer E2E tests to not share any of the app logic so that the E2E test is standalone and checks for values independently. In this case, if there were a bug in one of the helpers, your E2E test wouldn't also inherit that because it would just be looking for a raw string or DOM element. I personally prefer that too, but I think it's up the author's opinion.

Reply:
Makes sense! How should I test for those values then on the dom do you think?

Reply:
The same way you're doing it with the `data-e2e` attributes or by hardcoding a string value based on the static data that you know should be visible. It creates some maintenance if the helper changes, but I think that's okay

Reply:
Updated! 🙂